python/ray/function_manager.py

```diff
@@ -377,8 +377,9 @@ def fetch_and_register_remote_function(self, key):
              "driver_id", "function_id", "name", "function", "num_return_vals",
              "module", "resources", "max_calls"
          ])
-        function_id = ray.ObjectID(function_id_str)
-        driver_id = ray.ObjectID(driver_id_str)
+        function_id = ray.FunctionID(function_id_str)
+        # TODO(suquark): Make sure the type of "driver_id".
```

Comment:
This is the right type, do you want to remove the TODO?

JobID and DriverID are the same, and in the future I'd say we should transition to using DriverID exclusively. But given that in C++ it's called JobID, I think it's ok to call it JobID in python too until we change it everywhere.

Reply:
Yeah, we actually use both `DriverID` and `JobID` in C++. I agree about using only one (e.g., `DriverID`).